src/dosbox.cpp

```diff
@@ -985,6 +985,26 @@ const char *filter_on_or_off[] = {"on", "off", 0};
 	Pint->SetMinMax(0,100);
 	Pint->Set_help("the percentage of motion to ignore. 100 turns the stick into a digital one.");
 
+	Pbool = secprop->Add_bool("use_calibration_hotkeys", when_idle, false);
```

Comment:
Too generic, no? `use_joy_calibration_hotkeys` would be better.

Reply:
Indeed; good catch.
We might have other things that need runtime calibration (I can see mouse acceleration/sensitivity being something that can benefit from runtime adjustment)